doc/number.md

```diff
@@ -36,8 +37,8 @@ Napi::Number(napi_env env, napi_value value);
 ```cpp
 Napi::Number Napi::Number::New(napi_env env, double value);
 ```
- - `[in] env`: The `napi_env` environment in which to construct the `Napi::Nuber` object.
- - `[in] value`: The `napi_value` which is a handle for a JavaScript `Number`.
+ - `[in] env`: The `napi_env` environment in which to construct the `Napi::Number` object.
+ - `[in] value`: The C++ primitive from which to instantiate the `Napi::Value`.
```

Comment:
"... from which to instantiate the &#96;Napi::Number&#96;", right, because the return value is `Napi::Number`?

Reply:
In the string page it says Napi::Value, but I changed it. Maybe we need to edit the string docs as well?

Reply:
@heynemann we may indeed need to edit the string docs as well.

Reply:
However, it's OK if we do that in a second PR.

Reply:
Any additional feedback for this PR?